Integrations/integration-Windows_Defender_Advanced_Threat_Protection.yml

```diff
@@ -595,7 +645,7 @@ script:
             alert_severity = alert['severity']
             if alert_creation_time > last_alert_fetched_time and alert_status in FETCH_STATUS and alert_severity in FETCH_SEVERITY:
                 alert_id = alert['id']
-                alert['RelatedFiles'] = get_alert_related_files(alert_id)
+                #alert['RelatedFiles'] = get_alert_related_files(alert_id)
```

Comment:
why you commented this line?

Reply:
This functionality seem to return unexpected results in some cases and requires further investigation. Will be handled in the next enhancement.